maint-lib/stage.py

```diff
@@ -45,10 +45,26 @@
     sys.exit(1)
 
 
+def populateCephPointRelease():
+    # We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
```

Comment:
Turn into proper pydoc using triple-quotes
```py
"""
Extract Ceph version specifics from `CEPH_VERSION="version-point.release.1.2.3-1"
Sets 'CEPH_VERSION_NAME=version` and `CEPH_POINT_RELEASE='-point.release.1.2.3-1'`
"""

Reply:
This also reminds me that we should make sure we handle the case where the point release text has dashes in it.